courses/management/commands/sync_courseruns.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
Done, I also altered `--run` to be mutually exclusive with a new `--all` argument. That way the `--all` behavior is explicit rather than implicit, especially with the new command name.